R/filters.R

```diff
@@ -117,3 +119,31 @@ filter_by_repertoire <- function(.data, .min_size) {
 filter_by_clonotype <- function(.data, .query) {
   return(list(data = .data$data, meta = .data$meta))
 }
+
+include <- function(...) {
+  args <- unlist(unname(list(...)))
+  if (length(args) == 0) {
+    stop("include() expects at least 1 argument!")
+  }
+  return(c("include", args))
+}
+
+exclude <- function(...) {
+  args <- unlist(unname(list(...)))
+  if (length(args) == 0) {
+    stop("exclude() expects at least 1 argument!")
+  }
+  return(c("exclude", args))
+}
+
+lessthan <- function(value) {
+  return(c("lessthan", unname(value)))
+}
+
+morethan <- function(value) {
+  return(c("morethan", unname(value)))
+}
+
+interval <- function(from, to) {
+  return(c("interval", unname(from), unname(to)))
```

Comment:
you cannot do this because you can concatenate in vector only values of the same type, 'interval' and integers will cause error.

Reply:
If you need to include function name, you can use something like return(list(cmd='interval',c(from,to)))

Reply:
That's not big difference: from single vector it can be parsed as
```
query_type <- column_query[[1]]
query_args <- column_query[-1]
```